src/components/Cover/Cover.module.css

```diff
@@ -0,0 +1,68 @@
+.wrapper {
+    background-image: url("../../images/coverOverlay.svg"), url("../../images/coverPhoto.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+    min-height: 200px;
+    width: 100%;
```

Comment:
Section is a block and block byb default has 100% width

Reply:
fixed

Reply:
You should remove `width: 100%`, right?